pkg/lca/imagebasedupgrade.go

```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithAdditionalImages adds additionalImages to be used by the imagebasedupgrade.
+func (builder *ImageBasedUpgradeBuilder) WithAdditionalImages(
+	additionalImagesConfigMapName, additionalImagesConfigMapNamespace string) *ImageBasedUpgradeBuilder {
```

Comment:
Should we validate that string parameters are not empty?

Reply:
No need, patching with empty results in:

spec:
  additionalImages:
    name: ""
    namespace: ""